src/sqlancer/mysql/gen/MySQLTableGenerator.java

```diff
@@ -96,6 +96,8 @@ private void addCommonErrors(ExpectedErrors list) {
         list.add("Too many keys specified; max 1 keys allowed");
         list.add("The total length of the partitioning fields is too large");
         list.add("Got error -1 - 'Unknown error -1' from storage engine");
+        list.add(
```

Comment:
I think it would help to extract the error message and put them into a static field in the expected errors class for MySQL adding a comment that this is something that can happen on Windows.